nexus/README.md

```diff
@@ -0,0 +1,96 @@
+# nexus-hello-api
```

Comment:
Can we include a overview of what nexus is or link out to something that explains what this nexus is?

Reply:
And explain what the sample is showing